locallib.php

```diff
@@ -149,7 +150,8 @@ function local_boostnavigation_build_custom_nodes($customnodes, navigation_node
                             // Only proceed if something is entered here. This parameter is optional.
                             // If no language is given the node will be added to the navigation by default.
                             $nodelanguages = array_map('trim', explode(',', $setting));
-                            $nodevisible &= in_array(current_language(), $nodelanguages);
+                            $nodelanguage = in_array(current_language(), $nodelanguages);
+                            $nodevisible &= $nodelanguage;
```

Comment:
Is there a reason that you did split this line, but effectively don't change the functionality?

Reply:
OK, I see. This line is connected with your change in line 106.